### PR TITLE
refactor:add verification plugin checker

### PR DESCRIPTION
**Pull request by chloeyin**

### CODE REVIEW

File: verification/verifier_helpers.go [hunk 1]
```diff
@@ -13,6 +15,10 @@ import (
 	"github.com/notaryproject/notation-go/registry"
 )
 
+var errExtendedAttributeNotExist = errors.New("Extend attribute not exist")
```
Comment (JeyJeyGao) [inline verification/verifier_helpers.go:18]:
```suggestion
var errExtendedAttributeNotExist = errors.New("Extended attribute not exist")
```

File: verification/verifier.go [hunk 2]
```diff
@@ -151,8 +151,12 @@ func (v *Verifier) processSignature(ctx context.Context, sigBlob []byte, sigMani
 
 	// check if we need to verify using a plugin
 	var pluginCapabilities []plugin.Capability
-	verificationPluginName := GetVerificationPlugin(outcome.SignerInfo)
-	if verificationPluginName != "" {
+	verificationPluginName, err := getVerificationPlugin(outcome.SignerInfo)
+	// use plugin, but getPluginName returns an error
+	if err != nil && err != errExtendedAttributeNotExist {
+		return err
+	}
```
Comment (patrickzheng200) [inline verification/verifier.go:158]: we should also check VerificationPluginMinVersion by calling the getVerificationPluginMinVersion func here.

Comment (chloeyin) [reply verification/verifier.go:158]: > we should also check VerificationPluginMinVersion by calling the getVerificationPluginMinVersion func here.

Updated. PTAL

**Review by patrickzheng200 — APPROVED** [review]
lgtm

**Pull request merged**